content/registries/_index.md

```diff
@@ -13,3 +13,49 @@ weight: 23
 oc patch image.config.openshift.io/cluster -p \
 '{"spec":{"allowedRegistriesForImport":[{"domainName":"my.own.registry.example.com:8888","insecure":true}],"registrySources":{"insecureRegistries":["my.own.registry.example.com:8888"]}}}' --type='merge'
 ```
+
+# Configure custom/insecure registry to search path
+
+There seems to be no way to remove/modify or append any additional entries to unqualified-search-registries line in /etc/containers/registries.conf file as of now (OCP 4.3).
+
+There is way to modify the file using machineconfig which gets overwritten by editing `image.config.openshift.io/cluster` which will execute `99-master/worker-<uuid>-registries` and bring back the `unqualified-search-registries` again to the file, so this isn't the recommended way to perform the change. So to avoid to modify that CR, the machineconfig do the job that should be done by `image.config.openshift.io/cluster`. 
```

Comment:
I don't understand this :sweat_smile: can you rephrase it please?

Reply:
At this moment, there seems to be no way to remove/modify or append any additional entries to unqualified-search-registries line in /etc/containers/registries.conf file as of now (OCP <=4.3).

A workaround to modify the file is using a machineconfig that do the job that should be done by `image.config.openshift.io/cluster` (full control of the /etc/containers/{policy.json,registries.conf} content).

Yo have to be aware that it could gets overwritten by editing `image.config.openshift.io/cluster` which will execute `99-master/worker-<uuid>-registries` and bring back the `unqualified-search-registries` again to the file, so you must avoit to use the `image.config.openshift.io/cluster` afeter you applied the machineconfig.

Reply:
Could you please add those changes to the PR so I can merge it? Thanks